src/components/GuestList.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<NcSettingsSection :name="t('guests', 'Guests accounts')">
-		<div v-if="loaded && !error">
+		<div id="guests-list" v-if="loaded && !error">
```

Comment:
```suggestion
		<div class="guests-list" v-if="loaded && !error">
```
Let's use classes instead ;)
The style declaration will need to be changed to `.guests-list` as well